zetaclient/evm/utils.go

```diff
@@ -0,0 +1,159 @@
+package evm
```

Comment:
We should avoid using `utils` name as not very descriptive.

Can we maybe add these functions in other files.

Example `GetInboundVoteMsgForDepositedEvent` -> inbounds.go